docs/docusaurus/docs/reference/expectations/conditional_expectations.md

```diff
@@ -16,7 +18,7 @@ In Spark and SQLAlchemy, the `row_condition` value is parsed as a data filter or
 
 ## Examples
 
-To test if different encodings of identical pieces of information are consistent with each other, you run code similar to this example:
+To test if different encodings of identical pieces of information are consistent with each other, run a command similar to this example:
 
 ```python
```

Comment:
Another code block to flag as "should be pulled from an example script under test."

Reply:
@Rachel-Reverie this has been noted.